docs/Installation/topics/Installing_Zenko.rst

```diff
@@ -129,7 +129,10 @@ Follow these steps to install Zenko with Ingress.
    “:doc:`configure_ingress`” for additional terms to add to this chart.
 
 3. If your Zenko instance is behind a proxy, add the following lines to the
-   options.yml file, entering your proxy’s IP addresses and port assignments:
+   options.yml file, entering your proxy’s full IP address/URL and port 
```

Comment:
I think `proxy server address` makes more sense than saying IP/URL/address/locator to be consistent. Something like `entering your proxy's full server address including protocol and port` would work.